cloud_functions/shared/logger_utils.py

```
@@ -37,3 +39,31 @@ def setup_logger(name: Optional[str] = None, level: int = logging.DEBUG) -> Logg
+        message: The main log message
+        **kwargs: Additional key-value pairs to include in the log
+    """
+    log_data = {
+        "function": function_name,
+        "message": f"{function_name}-{message}",
+        **kwargs,
+    }
+    logger.info(json.dumps(log_data))
+
+
+def create_structured_logger(function_name: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_structured_logger`

(ANN201)